.github/workflows/buildpackage-mac.yml

```diff
@@ -34,15 +34,27 @@ jobs:
     - name: Make pyinstaller spec
       run: |
         pyi-makespec --hidden-import="pkg_resources.py2_warn" -F --add-data images/\*:images --add-data \*.png:. --add-data \*.ico:. -w -i P-face.icns pronterface.py
+        # Edit spec file
+        export git_hash=$(git rev-parse --short "$GITHUB_SHA")
+        sed -i '' '$ s/.$//' pronterface.spec
+        cat >> pronterface.spec <<EOL
+        ,
+        info_plist={
+            'CFBundleShortVersionString': '$git_hash',
+            'NSPrincipalClass': 'NSApplication',
+            'NSAppleScriptEnabled': False,
+            'NSAppSleepDisabled': True,
+          },
+        )
+        EOL
     - name: Make pyinstaller build
       run: |
         pyinstaller --clean pronterface.spec -y
-        ls -l dist/pronterface.app/Contents/macOS/pronterface
-        chmod +x dist/pronterface.app/Contents/macOS/pronterface
```

Comment:
Is this no longer needed?

Reply:
Nope! The app generated by Pyinstaller has execute permission as is, it doesn't have to be modified. We figured out that it was actually the stock Github `actions/upload-artifact` script zipping the app that messed with the file permissions. That problem was fixed when you changed the Actions script to zip the app manually in #1117, so this `chmod` is redundant!

Reply:
Awesome, one less workaround to maintain!